### ISSUE TITLE
A11y_code.visualstudio.com_DocsSetup_MacOS_Usability: Missing date format for keyboard only users at the end of the screen.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-docs#2807]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

URL: [https://code.visualstudio.com](https://code.visualstudio.com) 
Windows Version: Windows10
Browser Version: 
Microsoft Edge 42.17134.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1

**Repro Steps:**
Navigate to the url: [https://code.visualstudio.com/docs/setup/mac](https://code.visualstudio.com/docs/setup/mac) 
follow below steps:
1.	Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com) 
2.	Navigate to "Docs" link.
3.	From left navigation, select "Setup".
4.	Navigate to "MacOS" link.
5.	Start screen reader.
6.	Navigate to the bottom of the screen
7.	Navigate to the date provide and listen how it is announced.

**Actual:**
At the bottom of the screen date has been mentioned as "1/2/2019" but no any instructions are available regarding to date format as "DDMMYYYY" for keyboard only users.

This issue is observed with "Edge + Narrator" and "Chrome + NVDA".

**Expected:**
Date format should be available to the keyboard only users for the date present on the screen.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
The keyboard only users find it difficult to understand the date content according to a particular date format.

**Attachment for Reference:**
[Code.visualstudio.com_DocsSetup_MacOS_Usability Missing date format.pptx](https://github.com/microsoft/vscode-docs/files/3324778/Code.visualstudio.com_DocsSetup_MacOS_Usability.Missing.date.format.pptx)

**Comment by MayuriNaik** [comment]
Similar issue is observed in the following scenarios:
1) Themes - [https://code.visualstudio.com/docs/getstarted/themes](https://code.visualstudio.com/docs/getstarted/themes)
2) Continuous Integration - [https://code.visualstudio.com/api/working-with-extensions/continuous-integration](https://code.visualstudio.com/api/working-with-extensions/continuous-integration)
3) Display Language - [https://code.visualstudio.com/docs/getstarted/locales](https://code.visualstudio.com/docs/getstarted/locales)
4) Angular Tutorial - [https://code.visualstudio.com/docs/nodejs/angular-tutorial](https://code.visualstudio.com/docs/nodejs/angular-tutorial)
5) Java - [https://code.visualstudio.com/docs/languages/java](https://code.visualstudio.com/docs/languages/java)

**Comment by gregvanl** [comment]
Closing for now since Usability issues, labelled A11yUsable, are not currently required for Grade C.
